composer.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "mantisbt/mantisbt",
+    "description": "Mantis Bug Tracker",
+    "type": "project",
+    "require": {
+        "slim/slim": "^3.0"
+    },
+    "license": "GPL v2",
+    "authors": [
+        {
+            "name": "Victor Boctor",
+            "email": "vboctor@gmail.com"
```

Comment:
No offense, but you are not the only author of this project. IMO, this should be the same as we have in the headers, i.e. 

    "name": "MantisBT Team",
    "email": "mantisbt-dev@lists.sourceforge.net"